core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -105,8 +105,8 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
 
       container match {
         case Some((actor, data)) =>
-          busyPool.update(actor, data)
-          freePool.remove(actor)
+          busyPool = busyPool + (actor → data)
```

Comment:
We don't use ASCII arrows in our codebase.

Reply:
Fixing; was the default setup for my editor. longterm, having a `scalafmt` style is probably a good idea. I know IntelliJ can import it, suspect Eclipse, etc can as well.